bootstrap/sql/postgresql/v001__create_tables.sql

```diff
@@ -39,8 +39,9 @@ CREATE TABLE IF NOT EXISTS widget (
   "type" VARCHAR(255) NOT NULL,
   "data" TEXT NOT NULL,
   "timestamp"  BIGINT,
+  "dashboardId" BIGINT NOT NULL,
```

Comment:
If the user already has "widget" entity in this database, running this migration won't reflect changes to the column "dashboardId" as the table is created with "create if not exist" syntax. Can we add another sql migration script to modify dashboardId ?

Reply:
Yes you're right. Maybe need to have one another sql for postgre or fix them all from v004.